docs/0-installation.md

```diff
@@ -48,6 +48,9 @@ The generator adds these core files, among others:
 * `app/assets/stylesheets/active_admin.scss`
 * `config/initializers/active_admin.rb`
 
+If you do not want to require the javascript for ActiveAdmin on all pages,
+then make sure that you don't have `require_tree .` in your application.js
+
```

Comment:
We now have #6676 which includes a note for this case in the installer itself, so I'll close this in favour of that change, since I didn't get any feedback here.